src/generic/grid.cpp

```diff
@@ -8726,6 +8885,19 @@ void wxGrid::SetColLabelSize( int height )
     }
 }
 
+void wxGrid::SetRowLabelMinimalSize(int width)
+{
+    if ( width >= 0 )
```

Comment:
Does it really make sense to pass negative width here? If not, we should replace this with

```suggestion
    wxCHECK_RET( width >= 0, "invalid min row label width" );
```